commit 253b964f58a95c07c99d6302e2edf8ff17f33943
Author: Fedora Kernel Team <kernel-team@fedoraproject.org>
Date:   Thu Aug 3 13:46:51 2017 -0500

    UBUNTU: SAUCE: (efi-lockdown) Fix for module sig verification
    
    (cherry picked from commit 097bdd200fc81d90bbd955eb8e1010b453e7f546
     git://git.kernel.org/pub/scm/linux/kernel/git/jwboyer/fedora.git)
    Signed-off-by: Seth Forshee <seth.forshee@canonical.com>

diff --git a/kernel/module_signing.c b/kernel/module_signing.c
index 937c844bee4a..d3d6f95a96b4 100644
--- a/kernel/module_signing.c
+++ b/kernel/module_signing.c
@@ -81,6 +81,6 @@ int mod_verify_sig(const void *mod, unsigned long *_modlen)
 	}
 
 	return verify_pkcs7_signature(mod, modlen, mod + modlen, sig_len,
-				      NULL, VERIFYING_MODULE_SIGNATURE,
+				      (void *)1UL, VERIFYING_MODULE_SIGNATURE,
 				      NULL, NULL);
 }
